### ISSUE TITLE
'file_server sets `Date` header correctly' test case is flaky

### ISSUE THREAD

**Issue by kt3k** [denoland/deno_std#1058]

**Comment by getspooky** [comment]
@kt3k I confirm this issue , I run mutilple times `file_server_test.ts` ,the 3 first tests has been successfully passed but the last has failed.
```sh
failures:

file_server sets `Date` header correctly
AssertionError: Values are not equal:


    [Diff] Actual / Expected


-   903909
+   903910

```

**Comment by kt3k** [comment]
fixed by #1059

**Comment by uki00a** [comment]
This test still seems flaky :scream: (https://github.com/denoland/deno_std/pull/1094/checks?check_run_id=3242259868)
